### PR TITLE
fix(virtualRepeat): sets initial size for virtual repeat when the fir…

### LINKED ISSUE #5826: md-autocomplete in md-dialog have huge height also with only few values
Hi,
I'm using md-autocomplete inside md-dialog and the md-virtual-repeat-container that shows the possible choises is too high when I have only one element.
It's the right behaviour?

Thanks
schermata 2015-11-19 alle 12 50 32

### PR DESCRIPTION
**Pull request by robertmesserle**
…st results require shrinking

closes #5826

### CODE REVIEW

**Comment by robertmesserle** [comment]
@jelbourn @kseamon

File: src/components/virtualRepeat/virtual-repeater.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ VirtualRepeatContainerController.prototype.sizeScroller_ = function(size) {
 VirtualRepeatContainerController.prototype.autoShrink_ = function(size) {
   var shrinkSize = Math.max(size, this.autoShrinkMin * this.repeater.getItemSize());
   if (this.autoShrink && shrinkSize !== this.size) {
-    if (shrinkSize < (this.originalSize || this.size)) {
+    if (shrinkSize < (this.originalSize || this.size) || !(this.originalSize || this.size)) {
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: Needs a unit test

Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: Would it work if it was just 
```
if (shrinkSize < (this.originalSize || this.size) || !(this.originalSize))
```
?

Comment (kseamon) [inline src/components/virtualRepeat/virtual-repeater.js]: That does have a different meaning, but the way that looks is still pretty awkward.

**Pull request closed without merging**